src/System.Text.Http.Parser/BadHttpRequestException.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+
+namespace System.Text.Http.Parser
+{
+    internal static class StatusCodes
+    {
```

Comment:
No it's internal. I dunno if we even have a System.Net dependency when we wrote it. It's just s bunch of ints so it doesn't matter anyways.